src/services/services.ts

```diff
@@ -7530,7 +7533,7 @@ namespace ts {
         function walk(node: Node) {
             switch (node.kind) {
                 case SyntaxKind.Identifier:
-                    nameTable[(<Identifier>node).text] = (<Identifier>node).text;
+                    nameTable[(<Identifier>node).text] = nameTable[(<Identifier>node).text] === undefined ? node.pos : -1;

```

Comment:
Isn't this too order-dependent? What happens if you add code *prior* to another usage? For instance, I start out with:

```ts
if (Person.blah) {
    doSomething();
}
```

I'd want to also get completion for if I added a use of `Person.blah` before the `if` statement:

```
Person.blah/**/

if (Person.blah) {
    doSomething();
}
```